src/patternfly/components/InputGroup/input-group.scss

```diff
@@ -1,4 +1,8 @@
 .pf-c-input-group {
+  // Input Group
+  --pf-c-input-group--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
```

Comment:
can you use `var(--pf-global--BackgroundColor--100);` here? `@include pf-t-light;` in this component should ensure that's white in light/dark mode.